accord-core/src/test/java/accord/utils/MessageTask.java

```diff
@@ -88,7 +97,8 @@ public TaskRequest(NodeProcess process, String desc)
         @Override
         public void process(Node on, Node.Id from, ReplyContext replyContext)
         {
-            process.process(on, from, success -> on.reply(from, replyContext, success ? SUCCESS : FAILURE));
+            // TODO (review): This is kind of odd, no error to propagate?
+            process.process(on, from, success -> on.reply(from, replyContext, success ? SUCCESS : FAILURE, null));
```

Comment:
only happens in `accord.burn.TopologyUpdates#notify` ```long nodeEpoch = node.epoch();
            if (nodeEpoch + 1 < update.epoch())
                onDone.accept(false);```
                
                in this single case, an exception at least gives more detail; which is nice... hit this condition earlier this morning so a human readable string would be nice